tests/acceptance/api_test.go

```diff
@@ -397,17 +398,17 @@ func TestApi(t *testing.T) {
 			require.NotNil(t, data["lastModifiedOn"])
 			require.NotNil(t, data["leaseStatusModifiedOn"])
 
-			// Create the Decommission Request Body
-			body = leaseRequest{
-				PrincipalID: principalID,
-				AccountID:   acctID,
-			}
+			// Account should be marked as status=Leased
+			waitForAccountStatus(t, apiURL, acctID, "Leased")
```

Comment:
new test coverage, to verify account status update

Added a `waitForAccountStatus` helper util, since this is a common assertion in our tests